release/publish_release.sh

```diff
@@ -110,6 +110,10 @@ while getopts cd:e:g:h:i:k:l:mn:o:qr:t:u:v:wxyz arg ; do
   esac
 done
 
+if [[ "${DOCKER_DEST}" == "<none>" ]]; then
```

Comment:
It'd be useful to doc this under usage() for -z or -d, but I won't insist on it (esp since I didn't doc the empty-string option).  This check could be added as an "or" case to the "if [[ -z "${DOCKER_DEST}" ]]; then" around line 190 (just above the "NOTE: An empty string was used..."), but I suppose "none" is more explicit/intentional, and this location allows the "all operations ... disabled" check that follows to catch this corner case so I'm okay with either location.

Reply:
Not only is the cleaner but i implemented it as setting DOCKER_DEST before DOCKER_DEST is used so we know it will always have the same behavior.